README.rst

```diff
@@ -228,9 +228,8 @@ Development install
     source venv3/bin/activate
 
     # (from the project root directory)
-    # install howfairis as an editable package
-    pip install --no-cache-dir --editable .
-    pip install --no-cache-dir --editable .[dev]
+    # install howfairis as an editable package with its development dependencies
+    pip install --editable .[dev]
```

Comment:
I find the explicit inclusion one clearer

Reply:
ok changed it back.